### PR TITLE
Add BIOS (Merged,Non-Merged)

### PR DESCRIPTION
**Pull request by lasers**
BIOS (Merged).dat
* Everything goes in one zip. Perfect for RetroArch.

BIOS (Non-Merged).dat
* Separate system zips. Core friendly. No system core, no system zip.

Fixes https://github.com/libretro/libretro-database/issues/336 too.

### CODE REVIEW

**Comment by RobLoach** [comment]
Could we then remove BIOS.dat?

**Comment by lasers** [comment]
I can make a removal commit if you're satisfied.

File: dat/BIOS (Non-Merged).dat [hunk 1]
```diff
@@ -113,24 +105,21 @@ game (
 	rom ( name st011.data.rom size 4096 crc 803256a7 md5 10bd3f4aa949737ab9836512c35bcc29 sha1 798ab84110ca93a6e73a540f4146d21f8d9e1442 )
 	rom ( name st011.program.rom size 49152 crc a741e9fb md5 95222ebf1c0c2990bcf25db43743f032 sha1 375eca92592621e83881f3a9d7fd5ea3e5b306dc )
 	rom ( name st018.data.rom size 32768 crc b5255459 md5 49c898b60d0f15e90d0ba780dd12f366 sha1 b19c0f8f207d62fdabf4bf71442826063bccc626 )
-	rom ( name st018.program.rom size 131072 crc f73d5e10 md5 dda40ccd57390c96e49d30a041f9a9e7 sha1 388e3721b94cd074d6ba0eca8616523d2118a6c3 )
-	rom ( name BS-X.bin size 1048576 crc f51f07a0 md5 fed4d8242cfbed61343d53d48432aced sha1 604556b2e62860af18db5a77f2956ebc75450020 )
```
Comment (RobLoach) [inline dat/BIOS (Non-Merged).dat]: BS-X.bin is used in SNES9X....
https://github.com/libretro/snes9xherd/blob/0d34187679216d3cb96eed9c862958c7f6eeb14f/src/bsx.c#L966

File: dat/BIOS (Non-Merged).dat [hunk 2]
```diff
@@ -1,62 +1,52 @@
...
 	homepage "http://github.com/libretro/libretro-database"
 )
 
 game (
-	name "The 3DO Company - 3DO"
-	description "The 3DO Company - 3DO"
-	rom ( name goldstar.bin size 1048576 crc b6f5028b md5 8639fd5e549bd6238cfee79e3e749114 sha1 c4a2e5336f77fb5f743de1eea2cda43675ee2de7 )
-	rom ( name panafz1.bin size 1048576 crc c8c8ff89 md5 f47264dd47fe30f73ab3c010015c155b sha1 34bf189111295f74d7b7dfc1f304d98b8d36325a )
-	rom ( name panafz10.bin size 1048576 crc 58242cee md5 51f2f43ae2f3508a14d9f56597e2d3ce sha1 3c912300775d1ad730dc35757e279c274c0acaad )
-	rom ( name sanyotry.bin size 1048576 crc d5cbc509 md5 35fa1a1ebaaeea286dc5cd15487c13ea sha1 b01c53da256dde43ffec4ad3fc3adfa8d635e943 )
```
Comment (RobLoach) [inline dat/BIOS (Non-Merged).dat]: Found the references to 3DO in MAME:
https://github.com/libretro/mame2014-libretro/blob/2e3943dd6e9bf6e4161704179d7af2fbbd313e92/src/mame/drivers/3do.c#L194-L204

**Comment by lasers** [comment]
Thanks for looking. I got my information from a bad source.

**Pull request merged**